C/common/string_utils.cpp

```diff
@@ -292,3 +294,56 @@ void StringEscapeQuotes(std::string& str)
 
 	}
 }
+
+/**
+ * Remove space at both ends of a string
+ */
+char *trim(char *str)
+{
+	char *ptr;
+
+	while (*str && *str == ' ')
+		str++;
+
+	ptr = str + strlen(str) - 1;
+	while (ptr > str && *ptr == ' ')
+	{
+		*ptr = 0;
+		ptr--;
+	}
+	return str;
+}
+
+/**
+ * Remove space at both ends of a string
+ */
+std::string StringLTrim(const std::string& str)
+{
+	string output;
+	size_t pos = str.find_first_not_of(" ");
+
+	if (pos == std::string::npos)
+		output = "";
+	else
+		output = str.substr(pos);
+
+	return (output);
+}
+
+std::string StringRTtrim(const std::string& str)
```

Comment:
I suspect you really wanted to call this StringRTrim

Reply:
fixed